.github/workflows/R-CMD-check.yaml

```diff
@@ -22,16 +20,18 @@ jobs:
       fail-fast: false
       matrix:
         config:
+          - {os: macos-latest,   r: 'release'}
```

Comment:
```suggestion
```

Looks like this will actually work for R-devel! I added a check for macOS just because it is the "standard" workflow, but it may be work for another time to figure this one out!

Reply:
Thanks @olivroy, let me try that R-devel and see, and we shall add the changes to the main PR after.

Reply:
Just to be clear, it seems that https://github.com/weecology/portalr/commit/049f67c9c963c28228dd8b20508860788444c2ed and https://github.com/weecology/portalr/commit/761423b3df17fc70c510bf7e3cfbf650a77b0f29 actually fixed the r-devel issue. (I edited my comment above to reflect this)